packages/apps/plugins/plugin-outliner/src/utils/parser.ts

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { Tree } from '@braneframe/types';
+import { TextObject } from '@dxos/client/echo';
+
+export const tryParseOutline = (outline: string): Tree.Item[] | undefined => {
```

Comment:
just parseOutline.
Is this just markdown? If not provide a reference for the format?
E.g., is this Loqseq specific?